test/test_external.py

```diff
@@ -1,18 +1,16 @@
 import os
 from tempfile import gettempdir
 
-from pyabc.external import R
 import pyabc
-
-
-r_file = "doc/examples/myRModel.R"
+import pyabc.external
 
 
 def test_r():
```

Comment:
may be good to include additional languages at some point, e.g., Morpheus (once it is working)

Reply:
yes! for morpheus, we will even have a complete extra file (currently in morpheus_interface branch), but it should kind of extend the general ExternalModel class. then also tests need to be added.

Reply:
some model in C ? or matlab? :no_good_man:

Reply:
matlab we probably can't install on travis, in C I don't have a model. You have one in Java, right? You might very much want to add that here :). I think you mentioned that we'll need to pass additional arguments then. Can you name a typical command line call that would be used? Then we can maybe add functionality to cover such setups.

Reply:
Yes, i was thinking about this example.
normally I call scala via:
`call(["/usr/bin/taskset","-c","0",(pathMwn+"/mvnw"), "scala:run", "\"-DaddArgs={}\"".format(params_argument)])`

Reply:
added the possibility to define custom arguments in addition now.